common/changes/@uifabric/fabric-website/redo_2018-05-30-04-52.json

```diff
@@ -2,8 +2,8 @@
   "changes": [
     {
       "packageName": "@uifabric/fabric-website",
-      "comment": "revert build change",
-      "type": "none"
+      "comment": "made sure the deps are resolved by package rather than relative paths",
```

Comment:
Uhh this might cause problems. You should avoid the deletion of the revert change file as part of your change here.. If you uncommit the deletion then merge master, it will show up as a new change file rather than a rename.

Reply:
Sure, easy enough. Just didn't think this warrants a shrinkwrap update of everything :)